src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -77,3 +77,45 @@ <h5>{% trans "Configuration problems" %}</h5>
         {% endfor %}
     </ul>
 {% endif %}
+
+{% if invalid_registration_backends %}
+    <h5>{% trans "Registration backends problems" %}</h5>
+    <ul>
+        <ul>
+        {% for failed_backend in invalid_registration_backends %}
+            <li>
+                <p>
+                    {% blocktranslate with name=failed_backend.config_name message=failed_backend.exception_message trimmed %}
+                        The configuration for plugin '{{ name }}' is invalid.<br>
+                        ({{ message }})<br>
+                    {% endblocktranslate %}
```

Comment:
I think you'd still want to report in which forms this plugin is used so that the recipient can properly judge the impact of this problem.